app/head.tsx

```diff
@@ -0,0 +1,9 @@
+export default function Head() {
```

Comment:
I think you want `metadata` / `generateMetadata` instead of a `head.tsx` file, right?